site/src/components/Footer.js

```diff
@@ -0,0 +1,177 @@
+//modules imports
+import React from "react";
+import styled from "styled-components";
+
+//mui import
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import { makeStyles } from "@material-ui/core/styles";
+
+//assets import
+import PatternsLogo from "../assets/images/brand/SVG/service-mesh-patterns-side_white.svg";
+import slack from "../assets/images/socialIcons/slack-light.svg";
+import docker from "../assets/images/socialIcons/docker-light.svg";
+import github from "../assets/images/socialIcons/github-dark.svg";
+import linkedin from "../assets/images/socialIcons/linkedin-light.svg";
+
+const Footerdiv = styled.footer`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 25rem;
+  background-color: black;
+  align-items: center;
+`;
+
+const Wrapperdiv = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 85%;
+  gap: 7rem;
+`;
+
+const Patternsdiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+  align-items: center;
+  justify-content: center;
+  width: 18rem;
+  height: 100%;
+`;
+
+const Resourcesdiv = styled.div`
+  height: 100%;
+  gap: 1rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Resourceslink = styled.div`
+  display: flex;
+  gap: 0.3rem;
+  flex-direction: column;
+`;
+
+const Maildiv = styled.div`
+  display: flex;
+  gap: 2rem;
+  flex-direction: column;
+  justify-content: center;
+`;
+
+const Copyrightdiv = styled.div`
+  display: flex;
+  width: 70%;
+  border-top: 2px solid white;
+  justify-content: space-between;
+`;
+
+const Icondiv = styled.div`
+  display: flex;
+  gap: 0.5rem;
+`;
+
+const Inputdiv = styled.div`
+  display: flex;
+  gap: 1rem;
+`;
```

Comment:
Move the CSS to a separate file named `footer.style.js` and use one single wrapper div instead of separating them for each section. Use classNames for each section instead.

Reply:
Resolved